src/DIRAC/ResourceStatusSystem/Service/PublisherHandler.py

```diff
@@ -1,37 +1,37 @@
-""" PublisherHandler
+"""
+PublisherHandler
 
 This service has been built to provide the RSS web views with all the information
 they need. NO OTHER COMPONENT THAN Web controllers should make use of it.
-
 """
 #  pylint: disable=no-self-use
 from datetime import datetime, timedelta
 
 # DIRAC
 from DIRAC import S_OK, gConfig, S_ERROR
 from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC.Core.Utilities.ObjectLoader import ObjectLoader
 from DIRAC.Core.Utilities.SiteSEMapping import getSEHosts, getStorageElementsHosts
 from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getSites, getSiteCEMapping
 from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers
 from DIRAC.ResourceStatusSystem.Client.ResourceStatusClient import ResourceStatusClient
-from DIRAC.ResourceStatusSystem.Utilities import Utils
-
-ResourceManagementClient = getattr(
-    Utils.voimport("DIRAC.ResourceStatusSystem.Client.ResourceManagementClient"), "ResourceManagementClient"
-)
 
 
 class PublisherHandlerMixin:
     """
     RPCServer used to deliver data to the web portal.
-
     """
 
     @classmethod
     def initializeHandler(cls, serviceInfoDict):
         """Initialization of client objects"""
         cls.rsClient = ResourceStatusClient()
```

Comment:
Also this could be loaded via ObjectLoader.

Reply:
Concerning this class, should I raise an exception or should I return S_ERROR ? It's not an `__init__` method, and it already returns S_OK but I didn't manage to find where this function is called.

Reply:
You should return `S_ERROR`.